app/Helpers/PaginatedList.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace backend.Helpers
+{
+    public class PaginatedList<T> : List<T>
+    {
+        public int CurrentPage { get; private set; }
+        public int TotalPages { get; private set; }
+        public int CurrentRecords { get; private set; }
+        public int TotalRecords { get; private set; }
+
+        public PaginatedList(List<T> items, int count, int pageNumber, int pageSize)
```

Comment:
czym się różni count od items.Count?

Reply:
Tym, że items.Count to liczność obecnej listy po paginacji a count to liczność całego zbioru